src/AutoMerge/guidelines.jl

```diff
@@ -332,11 +332,25 @@ function meets_breaking_explanation_check(data::GitHubAutoMergeData)
     # Look up PR here in case the labels are slow to be applied by the Registrator bot
     # which decides whether to add the BREAKING label
     pr = GitHub.pull_request(data.api, data.registry, data.pr.number; auth=data.auth)
-    return meets_breaking_explanation_check(pr.labels, pr.body)
+    return meets_breaking_explanation_check(pr.labels, pr.body, data.version)
 end
 
-function breaking_explanation_message(has_release_notes)
-    example_detail = """
+function breaking_explanation_message(has_release_notes, version::VersionNumber)
+    msg = if has_release_notes
+        """
+        This is a breaking change, but the release notes do not mention it. Please add a mention of the breaking change to the release notes (use the words \"breaking\" or \"changelog\").
+        """
+    else
+        """
+        This is a breaking change, but no release notes have been provided. Please add release notes that explain the breaking change.
+        """
+    end
+    if version < v"1.0.0"
+        msg *= """
+        Given this is a pre-v1.0.0 release, you may have not intended to make a breaking change release. [More information](https://pkgdocs.julialang.org/v1/compatibility/#compat-pre-1.0) on Julia's handling of pre-v1.0.0 versioning.
```

Comment:
I think we should make this more "actionable", like if this was intended do X, if it was not intended do Y